### PR TITLE
Touch up ManagedFileChooser design a bit

### PR DESCRIPTION
**Pull request by emmauss**
## What does the pull request do?
Cleans up ManagedFileChooser design a bit and fixes some issues with both Simple and Fluent designs. Changes include
1. Added a margin to the root DockPanel in Simple design
2. Set fixed widths to OK and Cancel button.
3. Fixes GridSplitters not detecting the correct direction in Simple Design
4. Remove GridSplitter for Icon column. It doesn't make sense having one there. Instead, added one for Size column.
5. Size column contents are aligned to the right, to match common behaviors in grid when displaying numerical values.

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035304-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

File: src/Avalonia.Themes.Fluent/Controls/ManagedFileChooser.xaml [hunk 1]
```diff
@@ -179,8 +179,8 @@
               </DockPanel>
               <CheckBox IsChecked="{Binding ShowHiddenFiles}" Content="Show hidden files" DockPanel.Dock="Left"/>
               <UniformGrid x:Name="Finalize" HorizontalAlignment="Right" Rows="1">
-                <Button Command="{Binding Ok}">OK</Button>
-                <Button Command="{Binding Cancel}">Cancel</Button>
+                <Button Command="{Binding Ok}" Width="80">OK</Button>
```
Comment (robloo) [inline src/Avalonia.Themes.Fluent/Controls/ManagedFileChooser.xaml:182]: Don't use fixed Width where text may change in the future. As soon as Avalonia supports localization this is going to have to be translated.

It is better to set a MinWidth in cases like this.

Comment (emmauss) [reply src/Avalonia.Themes.Fluent/Controls/ManagedFileChooser.xaml]: Switched to MinWidth

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0035358-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**